crates/swc_plugin/src/lib.rs

```diff
@@ -1,7 +1,10 @@
 // Reexports
-pub use swc_common::chain;
-pub use swc_ecma_ast::*;
-pub use swc_ecma_visit::*;
+pub use swc_common::{chain, DUMMY_SP};
+pub mod ast {
```

Comment:
this is small bikeshedding for the reexports namespace. Bit unsure this would be better or just having everything under top-level reexport would be better, would like to try couple of interatiions.

One gain for having namespace is author can to wildcard for the ast only like

```
swc_plugin::{ast:*, ...} 
```

instead of wildcard import every top level exports.

Reply:
Seems like a good idea. I love this :)